src/HornedBeast.js

```diff
@@ -1,19 +1,31 @@
 import React, { Component } from 'react';
- 
-
+import data from './data.json';
     class HornedBeast extends Component {
-        render(){
-            return(
-                <div>
-                    <h2>{this.props.title}</h2>
-                    <img src={this.props.img_url} alt={this.props.alt} title={this.props.title} />
-
-                    <p>{this.props.description}</p> 
-
-                </div>
+        constructor(props) {
+            super(props);
+            this.state = {
+              votes: 0,
+              beast: {}
+              beastData: data
```

Comment:
For line 9, I would think about what the beast data should be. Should it be props or state? The lab asks us to pass props to the hornedBeast component. But Main is taking care of that job already for us/or well the lab wants Main to be the component who passes those props :)

I would go ahead and remove the beastData/json data out of state since you are importing the file/data into main.